ui/ts/models/log.ts

```diff
@@ -58,39 +58,58 @@ module Models {
         true
       );
 
-      private _url(): string {
-        let url: string = "/_status/logs/";
+      /**
+       * buildURL creates the route and query parameters based on the current
+       * state of all the properties on entries.
+       * TODO(bram): consider exporting this to a utility function and maybe
+       * adding an interface for these top level models. Tamir also suggested
+       * moving to a single param with a tighter serialized format.
+       */
+      buildURL(): string {
+        let url: string = "/logs/";
```

Comment:
I did look into using that. But it looked like an extra tool that was just kind of thrown in there and barely documented.
URL.js would be a better choice I think.  Especially since it doesn't rely on jquery.